src/server/helpers.py

```diff
@@ -42,8 +45,14 @@ def render_template(template, *args, **kwargs):
         if os.path.isfile(year_lang_template):
             template_supported_years.append(y)
 
+    date_published = get_file_date_info(template, "date_published")
+    date_modified = get_file_date_info(template, "date_modified")
+    ebook_size_in_mb = get_ebook_size_in_mb(lang, year)
```

Comment:
I'm still confused why every page needs to know the ebook size in MB. Isn't this only needed by the TOC page?

Reply:
The ebook is now linked in the ToC menu which appears on every page. The size is needed for two reasons:
1. To decide whether to show the ebook or not. We don’t show in French 2019 for example as no ebook yet. Nor in any 2020 language - including English. Could have another flag but decided why have two when want size anyway.

2. As we link to the PDF (and I know you’ve raised #1517 to discuss this but personally I think we should leave as is) it’s rude to ask them to download a huge file without making them away of the size. And it doesn’t take up much space. So the size shows in the ToC menu in every page.